Remotion/Web/Development.WebTesting/HostingStrategies/Configuration/HostingConfiguration.cs

```diff
@@ -40,12 +40,16 @@ public class HostingConfiguration : IHostingConfiguration
         };
 
     private readonly ProviderSettings _hostingProviderSettings;
+    private TimeSpan _verifyWebApplicationStartedTimeout;
+
+    public TimeSpan VerifyWebApplicationStartedTimeout => _verifyWebApplicationStartedTimeout;

```

Comment:
According to the style guide, non-auto properties should be placed after the ctor.